backend/routes/postReaction.py

```
@@ -190,6 +190,12 @@ def rsvp_event(
     get_event_by_id(db, event_id)  # Ensure event exists
     return update_or_create_rsvp(db, event_id, current_user.id, attendee_data.status)
 
+@router.get("/event/{event_id}/my_rsvp/")
+def get_user_rsvp_status(event_id: int, db: Session = Depends(get_db), current_user: User = Depends(get_current_user)):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_user_rsvp_status`

(ANN201)